src/components/EmojiBar/EmojiBar.js

```diff
@@ -179,7 +179,7 @@ export default class AnimationScreen extends Component {
     // the margin top the box is 100
     // and plus the height of toolbar and the status bar
     // so the range we check is about 150 -> 450
-    if ((Platform.OS === 'android' && gestureState.y0 + gestureState.dy >= 450 && gestureState.y0 + gestureState.dy <= 600)
+    if ((Platform.OS === 'android' && gestureState.y0 + gestureState.dy >= 600 && gestureState.y0 + gestureState.dy <= 1000)
```

Comment:
keep this between 450 and 600 for now until we figure out how to make this responsive

Reply:
I used 600 because I can then test it on my phone

Reply:
all good, i fixed this issue in https://github.com/PubNubDevelopers/PubMoji/pull/22 

after merging this all devices should work